api/envoy/api/v2/endpoint/endpoint.proto

```diff
@@ -123,5 +123,5 @@ message LocalityLbEndpoints {
   // next highest priority group.
   //
   // Priorities should range from 0 (highest) to N (lowest) without skipping.
-  uint32 priority = 5;
+  uint32 priority = 5 [(validate.rules).uint32 = {gte: 0, lte: 128}];
```

Comment:
Thanks, this looks basically right. You don't need the `gte: 0` though, since this is an unsigned int. @alyssawilk what do you think a sensible upper bound here is? Will 128 work? Seems reasonable to me.